local/core/etc/config-settings/testnet.json

```diff
@@ -7,45 +7,45 @@
       "contract_compute_v0": {
```

Comment:
If pubnet is different to testnet, could we also add a pubnet.json?

Reply:
The only differences between pubnet and testnet at the moment are `bucket_list_target_size_bytes` and `write_fee1_kb_bucket_list_low`. The actual bucket list sizes will be quite low (and not the same as testnet or pubnet) when starting quickstart, so fees can't be calculated accurately. @dmkozh maybe we can come up with values where the fees will closely model testnet or pubnet with a very small bucket list?

Reply:
Testnet write fees are already kind of problematic as they fluctuate much more than pubnet write fees (just because, say, +20% BL growth may just be a few MB worth of data). This has recently been flagged by the users, as testnet fees are currently 3x higher than pubnet.

I'm not sure there is a great solution to this. Probably the least annoying thing we could do is to make write fees almost constant on testnet and make them much the current pubnet fee. The pubnet fee will change with time, but that should be much slower and we could maybe update the config every once in a while.